scripts/build-rhcc-image.sh

```diff
@@ -83,24 +97,54 @@ else
     INTERNAL_IMAGE_NAME=cb-rhcc/sync-gateway
 fi
 
+# Figure out whether which platforms we're targeting
```

Comment:
"whether which"

Also I think it'd be clearer to way "which *arches* we're targeting" here.

Reply:
This is no longer required in the most recent commit.